homeassistant/components/lightwave/sensor.py

```diff
@@ -0,0 +1,83 @@
+"""Support for LightwaveRF TRV - Associated Battery."""
```

Comment:
Do you even need these battery sensors? Since you are displaying and using the battery levle in climate.py already.

Reply:
Yes, as separate battery sensors they show much better in Lovelace.  Plus it’s a bit easier to create an automation when the battery is low.
If you are concerned about the duplication, I’d tend to remove it from the climate.py, but don’t see any harm in it being there twice.

Reply:
Understandable, normally I would recommend to remove the duplication, however since you are updating via `read_trv_status()` which fetches all the relevant data I think it is fine if it stays.